fdbcli/fdbcli.actor.cpp

```diff
@@ -3656,8 +3657,34 @@ ACTOR Future<int> runCli(CLIOptions opt) {
 		[](std::string const& line, std::vector<std::string>& completions) {
 			fdbcli_comp_cmd(line, completions);
 		},
-		[](std::string const& line)->LineNoise::Hint {
-			return LineNoise::Hint();
+		[enabled=opt.cliHints](std::string const& line)->LineNoise::Hint {
+			if (!enabled) {
+				return LineNoise::Hint();
+			}
+
+			bool error = false;
+			bool partial = false;
+			std::string linecopy = line;
+			std::vector<std::vector<StringRef>> parsed = parseLine(linecopy, error, partial);
+			if (parsed.size() == 0 || parsed.back().size() == 0) return LineNoise::Hint();
+			StringRef command = parsed.back().front();
+			int finishedParameters = parsed.back().size() + error;
+
+			// We don't want the hint to flip to parse error and back, e.g. while \" is being typed.
```

Comment:
I spent 1 min understanding the sentence here. 
I thought `back` is verb and later realized it is the "\" symbol.
Would you mind rewording this for clarity?